.github/workflows/actcheck.yaml

```diff
@@ -0,0 +1,27 @@
+name: actionlint
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches: [staging]
+    paths:
+      - '.github/workflows/**'
+
+jobs:
+  action-lint:
+    runs-on: ubuntu-latest
+    name: Action Lint
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: Install action linter
+        run: |
+          mkdir -p ~/bin
+          curl -sL https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash | bash -s -- latest ~/bin
+
+      - name: Check that all workflows are valid
+        run: |
+          set -xe
```

Comment:
not: `set -e` is the default, it's not necessary to set it again.

Reply:
didn't know this, nice

Reply:
I see you approved, I guess we can leave it in, doesn't hurt :sweat_smile:

Reply:
I approved with a nit pick, didn't want to block the PR on that.